contracts/libraries/UniswapLibrary.sol

```diff
@@ -488,7 +488,7 @@ library UniswapLibrary {
         require(
             xU3LP &&
                 amountOutReceived >
-                amountInSwapped.sub(amountInSwapped.div(SWAP_SLIPPAGE * 2)),
+                amountInSwapped.sub(amountInSwapped.div(SWAP_SLIPPAGE / 2)),
             "One inch swap slippage > 2 %"
```

Comment:
Don't we need to update the message here?

Reply:
No, because the slippage was 0.5% before - this fix corrects it to 2%, which it should have been.

Reply:
I see